lib/registration.pm

```diff
@@ -182,6 +183,23 @@ sub assert_registration_screen_present {
     }
 }
 
+sub verify_preselected_modules {
+    my $modules_needle = shift;
+
+    return if check_screen($modules_needle, 30);    # pre-selected modules visible without scrolling
```

Comment:
Can we use assert_screen with alternatives check here?

Reply:
It's tricky, I would say that it's not, as we had similar hack before as well. What we could do is assert that page is loaded and then have check_screen with 0 timeout

Reply:
We don't have AFIK any potential match to add it as an alternative when we run module addon_products_via_SCC_yast2 when the checkbox is checked and we cannot see the modules in one shot.